common/app/views/support/HtmlCleaner.scala

```diff
@@ -298,8 +298,17 @@ case class LiveBlogLinkedData(isLiveBlog: Boolean)(implicit val request: Request
         el.select(".block-time.published-time time").foreach { time =>
           time.attr("itemprop", "datePublished")
         }
-        el.select(".block-elements").foreach { body =>
-          body.attr("itemprop", "articleBody")
+        el.select(".block-title").foreach { title =>
+          title.attr("itemprop", "headline")
+
+        }
+        el.select(".block-elements").foreach { blockElements =>
```

Comment:
Good point - I added it in the commit message but it's not clear from the code itself. I'll add a note.